util/appset/appset.go

```diff
@@ -0,0 +1,23 @@
+package appset
+
+import (
+	"fmt"
+	"os/exec"
+
+	"github.com/argoproj/argo-cd/v2/pkg/apis/application/v1alpha1"
+	executil "github.com/argoproj/argo-cd/v2/util/exec"
+)
+
+// AppRBACName formats fully qualified application name for RBAC check
+func AppSetRBACName(appSet *v1alpha1.ApplicationSet) string {
+	return fmt.Sprintf("%s/%s", appSet.Spec.Template.Spec.GetProject(), appSet.ObjectMeta.Name)
+}
+
+func RunCommand(command v1alpha1.Command) (string, error) {
```

Comment:
What is this used for? And where?

Reply:
I was using this for my prior approach. I missed removing this part of the code. Will remove it, thanks.